crates/client/l1-messages/src/config.rs

```diff
@@ -0,0 +1,36 @@
+use std::fs::File;
+use std::path::PathBuf;
+
+pub const DEFAULT_ETHEREUM_NODE: &str = "127.0.0.1:8545";
+pub const DEFAULT_CONTRACT_ADDRESS: &str = "0x5fbdb2315678afecb367f032d93f642f64180aa3";
+
+#[derive(Clone, PartialEq, serde::Deserialize, Debug)]
+pub struct L1MessagesWorkerConfig {
+    #[serde(default = "default_http")]
+    pub http_provider: String,
+    #[serde(default = "default_contract_address")]
+    pub contract_address: String,
+}
+
+impl TryFrom<&PathBuf> for L1MessagesWorkerConfig {
+    type Error = String;
```

Comment:
For errors: string < anyhow < dedicated struct

I think for this we should at least use anyhow. But maybe having a dedicated struct will be better. Maybe it's not worth it, depending on the way this error is later handled

Reply:
I applied attitude from `data-availability` lib but will rewrite into `anyhow`